services/movie.py

```diff
@@ -22,6 +27,7 @@
     return Movie.objects.get(id=movie_id)
```

Comment:
The 'get_movie_by_id' function should handle the case where a movie with the given ID does not exist. Consider using 'get_object_or_404' or handling the 'DoesNotExist' exception to provide a more user-friendly error message.